Client/src/Components/product-overview/ProductOptions.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import StyleSelector from "./StyleSelector";
+import DataContext from "../context";
+
+const { useContext } = React;
+
+function ProductOptions() {
+  const data = useContext(DataContext);
+  const defaultPrice = data.product.default_price;
+  const { styles, styleIndex } = data;
+  const originalPrice = styles[styleIndex].original_price;
+  const salePrice = styles[styleIndex].sale_price;
+  const price = () => {
+    if (salePrice !== "0") {
+      return (
+        <span className="price">
+          <b>{salePrice}</b>
+          {" "}
+          <s>{originalPrice}</s>
+        </span>
+      );
+    } if (originalPrice !== "0") {
+      return (<span className="price">{originalPrice}</span>);
+    }
+    return (<span className="price">{defaultPrice}</span>);
+  };
+  const { category, name } = data.product;
+
+  return (
+    <div>
+      <div>STAR RATING PLACEHOLDER</div>
+      <span className="productCategory">{category}</span>
+      <h1>{name}</h1>
+      {price()}
+      <StyleSelector
+        styleIndex={styleIndex}
```

Comment:
I dont think you need to pass in styleIndex since you're getting styleIndex in StyleSelector from context.js